discovery-package/src/main/resources/bin/start.sh

```diff
@@ -136,6 +136,7 @@ _BPX_JOBNAME=${ZWE_zowe_job_prefix}${DISCOVERY_CODE} java -Xms32m -Xmx256m ${QUI
     -Dapiml.service.hostname=${ZWE_haInstance_hostname:-localhost} \
     -Dapiml.service.port=${ZWE_configs_port:-7553} \
     -Dapiml.discovery.staticApiDefinitionsDirectories=${ZWE_STATIC_DEFINITIONS_DIR} \
+    -Dapiml.discovery.serviceIdPrefixReplacer=${APIML_SERVICE_PREFIX_REPLACER} \
```

Comment:
The name needs to follow standard V2 format:
e.g. ZWE_components_gateway_server_ssl_enabled
In this case I would propose
ZWE_components_discovery_server_prefix_replacer

Reply:
Got it, I will fix it, thanks!